modules/brainbrowser/css/brainbrowser.css

```diff
@@ -1,194 +1,190 @@
 html {
-  overflow: scroll;
-  overflow: auto;
-  min-height: 100%;
-  margin-bottom: 1px;
+    overflow: auto;
```

Comment:
Is overflow: scroll; missing?

Reply:
@paiva The first overflow property is overwritten by the second. I removed the first because auto will overwrite the scroll value.